docs/detections/detections-exclude-cold-frozen-data-tiers.asciidoc

```diff
@@ -1,18 +1,24 @@
 [[exclude-cold-frozen-data-individual-rules]]
-== Exclude cold and frozen data from a rule
+== Exclude cold and frozen data from rule executions
 
 :frontmatter-description: Configure a rule to ignore cold and frozen data during execution. 
 :frontmatter-tags-products: [security]
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [manage]
 
-Rules that query cold and frozen data might perform more slowly. To exclude cold and frozen data, add a Query DSL filter that ignores cold and frozen {ref}/data-tiers.html[data tiers] when executing. You can add the filter when creating a new rule or updating an existing one. 
+Rules that query cold and frozen {ref}/data-tiers.html[data tiers] might perform more slowly. To exclude query results from cold and frozen tiers, add a Query DSL filter that ignores cold and frozen documents when executing. This can help Elasticsearch exclude cold and frozen data more efficiently. You can add the filter when creating a new rule or updating an existing one.
```

Comment:
Could we make it more clear and explicit that this workflow applies to **SINGLE** rules, not **ALL** rules?

Reply:
Good call, @peluja1012. How's this:

```suggestion
Your rule might perform slower if it queries data from cold and frozen {ref}/data-tiers.html[data tiers]. To help Elasticsearch efficiently exclude cold and frozen data, apply a Query DSL filter that ignores cold and frozen documents when your rule executes. You can add the filter when creating a new rule or updating an existing one.
```

cc: @yctercero @marshallmain

Reply:
Verified this change with @peluja1012 over slack.